x-pack/plugins/actions/server/sub_action_framework/validators.ts

```diff
@@ -42,3 +52,40 @@ export const buildValidators = <
     },
   };
 };
+
+const buildCustomValidators = <Config, Secrets>(
+  validators?: Array<ConfigValidator<Config> | SecretsValidator<Secrets>>
```

Comment:
Maybe we can create a type in `x-pack/plugins/actions/server/sub_action_framework/types.ts`